llvmlite/tests/test_binding.py

```diff
@@ -398,7 +398,7 @@ def test_set_option(self):
     def test_version(self):
         major, minor, patch = llvm.llvm_version_info
         # one of these can be valid
-        valid = [(10, 0), (9, 0)]
+        valid = [(11, 1), (9, 0)]
```

Comment:
Remove 9?

Reply:
done in 4706b5f19e5ae52fcac5e761088a9e0fa7c7bb77 --